src/compressed_tensors/quantization/lifecycle/initialize.py

```diff
@@ -79,6 +94,8 @@ def initialize_module_for_quantization(
         # no scheme passed and layer not targeted for quantization - skip
         return
 
+    _clear_all_qparams(module)
```

Comment:
When will this occur? if we re-apply quantization to the same module, shouldn't that be a user error?

Reply:
Discussed offline. I'm good with this change!